### PR TITLE
bind: Made config package containing the local configuration file nam…

### PR DESCRIPTION
**Pull request by krevsbech**
…ed.conf.local.

### CODE REVIEW

**Comment by kimrhh** [comment]
commit message style, you need to briefly describe the change in the first line, then add an empty line and then follow up with a longer description.

suggestion for line1: "bind: add config package'

File: recipes/bind/bind.inc [hunk 1]
```diff
@@ -54,6 +55,6 @@ inherit binconfig
 BINCONFIG_FILES = "${bindir}/isc-config.sh"
 FILES_${PN}-dev += "${bindir}/isc-config.sh"
 DEPENDS_${PN}-dev = "libssl libcrypto ${DEPENDS_LIBC}"
-
-PACKAGES =+ "${PN}-utils "
+FILES_${PN}-config = "${sysconfdir}/bind/named.conf.local"
```
Comment (kimrhh) [inline recipes/bind/bind.inc]: are there no other conf files?

Comment (krevsbech) [inline recipes/bind/bind.inc]: Yes there are a number of configs, apart from the .conf.local. However, as I see it we dot he conf package in order to allow later override to provide machine/project specific configuration. And for that purpose only named.conf.local i relevant.
Please consider that including more files also force config packages in project/machine to include the same files.

Comment (kimrhh) [inline recipes/bind/bind.inc]: that's fine, i just need to check that we are not still pushing some oe-lite default configuration package as part of the non-config packages.

**Comment by krevsbech** [comment]
Rewording in git log and added comment in recipe.

**Comment by kimrhh** [comment]
merged, thanks

**Pull request merged**